be/src/exec/broker_scanner.cpp

```diff
@@ -453,6 +454,16 @@ bool BrokerScanner::convert_one_row(
 
 // Convert one row to this tuple
 bool BrokerScanner::line_to_src_tuple(const Slice& line) {
+
+    if (!validate_utf8_fast(line.data, line.size)) {
```

Comment:
I think you can use a macro to use avx version in avx supported CPU

Reply:
OK